instrumentation/kafka-clients-0.11/src/main/java/io/opentelemetry/auto/instrumentation/kafkaclients/KafkaProducerInstrumentation.java

```diff
@@ -93,6 +93,11 @@ public static SpanWithScope onEnter(
 
       callback = new ProducerCallback(callback, span);
 
+      boolean isTombstone = record.value() == null && !record.headers().iterator().hasNext();
+      if (isTombstone) {
+        span.setAttribute("tombstone", true);
```

Comment:
This attribute is certainly not from semantic convention. Why should we want this into Otel?

Reply:
we (currently at least) capture lots of span attributes that are not defined by semantic conventions

maybe better if we prefixed this, e.g. `kafka.tombstone`, so it doesn't look like a semantic attribute?

Reply:
"We are already doing strange things, let us do more strange things" is not exactly good reason :) We already have unanswered questions "why do we have this attribute here", such as `dispatcher.target` or `servlet.origin`. I am hesitant to add more.

Reply:
i created #602 to track and make decision about this